src/themes/default/elements/icon.overrides

```diff
@@ -137,6 +148,16 @@ i.icon.add.to.cart:before { content: "\f217"; }
 i.icon.in.cart:before { content: "\f218"; }
 i.icon.add.user:before { content: "\f234"; }
 i.icon.remove.user:before { content: "\f235"; }
+i.icon.object.group:before { content: "\f247"; }
+i.icon.object.ungroup:before { content: "\f248"; }
+i.icon.clone:before { content: "\f24d"; }
+i.icon.hand.rock:before { content: "\f255"; }
```

Comment:
This new subset makes ref to "Rock-Paper-Scissors-Lizard-Spock". It's OK to put them under the User Actions category?